pkg/tasks/prerequisites.go

```diff
@@ -111,6 +111,30 @@ func createEnvironmentFile(s *state.State) error {
 	return err
 }
 
+func disableNMCloudSetup(s *state.State, node *kubeoneapi.HostConfig, _ ssh.Connection) error {
+	if node.OperatingSystem != kubeoneapi.OperatingSystemNameRHEL {
+		return nil
+	}
+
+	var allHosts = s.LiveCluster.ControlPlane
+	allHosts = append(allHosts, s.LiveCluster.StaticWorkers...)
+	for _, host := range allHosts {
+		if node.ID == host.Config.ID && !host.Initialized() {
+			cmd, err := scripts.DisableNMCloudSetup()
+			if err != nil {
+				return err
+			}
+
+			s.Logger.Infoln("Disable nm-cloud-setup...the node will be rebooted…")
```

Comment:
Please use three dots instead of three dots as a single character. We got some reports that it doesn't render well in the past.

```suggestion
			s.Logger.Infoln("Disable nm-cloud-setup... the node will be rebooted...")
```